xml/hwbp_registers.xml

```diff
@@ -549,6 +552,37 @@
         </field>
     </register>
 
+    <register name="External Trigger" short="exttrigger" address="0x7a1">
+        This register is accessible as \RcsrTdataOne when \FcsrTdataOneType is 7.
+
+        This trigger fires when any selected external debug trigger input
+        signals.  Up to 16 external
+        debug trigger inputs coming from other blocks within the hart,
+        (e.g. signaling an hpmcounter overflow) can be selected.  Hardware
+        may support none or just a few external debug trigger inputs
+        (starting with external debug trigger input 0 and continuing
+        sequentially).  Unsupported inputs are hardwired to be inactive.
```

Comment:
I wonder if we should use a name other than "external debug trigger" to avoid confusing these with the external triggers in the debug module referred to in dmcs2 and used for halt and resume groups.  Maybe something like intra-hart logic trigger?

Reply:
Sure.  Intra-hart logic trigger is fine with me.

There's the description but also the register name.  This register name should be similar to the description and should probably avoid using the same name as the dmcs2 exttrigger field (which it currently conflicts with).  What should this register be called?
"intrahartlogictrigger" is too long.
"ihlt" is too short.
"ihltrigger"?
"iharttrigger"?  (The unicode "I❤️trigger" is probably out. 😄)
"intrahart"?

Or maybe there's a description other than "intra-hart logic trigger" that lends itself to a more natural name.

Reply:
This is a tricky one. Intra-hart seems limiting. These triggers could be connected to something outside the RISC-V core altogether, and that should be fine.

They really are external triggers, going to a hart. The external triggers in dmcs2 are also external triggers, going to a Debug Module. Should we rename both? And then the direction matters, too.

dmcs2.exttrigger refers to input/output triggers to/from the DM.
exttrigger.select refers to input triggers to the hart.

I'm not arriving at a great name here, but just sharing some thoughts.